### ISSUE TITLE
Unable to get new InSpec provisioner working

### ISSUE THREAD

**Issue by lfarnell** [hashicorp/packer#7334]
cc @xinau

I checked out master branch and built it locally to test out the new InSpec provisioner but I was unable to make it work. It fails every time no matter what I try. I tried the example in the docs but this did not work so my template below is my modified version of it. Attached below are the packer template and the output logs.

I know that this uses an SSH Proxy but does this work on Windows? I assume it would but I just want to confirm.

- Packer version from `packer version`
1.3.5.-dev
- Host platform
Windows 10
- Debug log output from `PACKER_LOG=1 packer build template.json`.
https://gist.github.com/lfarnell/b71bbf43f978522052133e535666fe9a
- The _simplest example template and scripts_ needed to reproduce the bug.
https://gist.github.com/lfarnell/95c894c01362cd88cfd182a838e96759

**Comment by xinau** [comment]
@lfarnell Strange.
What is the output from the follwing command (The command is taken from your packer output, you might need to change it). 
```
inspec exec https://github.com/dev-sec/linux-baseline --backend ssh --host 127.0.0.1 --key-files C:\Users\lfarnell\AppData\Local\Temp\packer-provisioner-inspec.971778311.key --user root --port 64880 --attrs C:\Users\lfarnell\AppData\Local\Temp\packer-provisioner-inspec.572872378.yml
```

**Comment by lfarnell** [comment]
@xinau I apologize for the false bug report. I just ran it again when I got to work and IT WORKS 😃 

It looks like an [issue](https://stackoverflow.com/questions/39179377/windows-bundle-install-home-environment-variable-or-homedrive-and-homepath-mu) with Ruby on Windows. I was using a domain connected machine and the HOMEDRIVE Env variable is pointed at a UNC path and not C:\ drive so because I wasn't on the company network, it failed.

Here's the log if your interested.

https://gist.github.com/lfarnell/1f6a6cc3ac3eea767a52e139ef6ce8f5
